index.html

```diff
@@ -56,14 +58,11 @@ <h2><span class="flex">experience:</span></h2>
            <li><a href="#leave-message">Leave a Message</a></li>
          </ul>
          </nav>
-    </header>
-     <main>
-     </main>
      <section id="messages">
       <h2><span class="flex">Messages</span></h2>
-      <div id="content">
-        <ul></ul>
-      </div>
+      <div id="content"></div>
+    </main>
```

Comment:
It's not working with `</main>` here, so you'll just want to move it after what you consider the end of your "main content", but it cannot be inside of a section. Your indents are not consistent, which makes it tough to see where sections start and end, so updating those could really help too.